<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NiveauTraceModules</class>
 <widget class="QDialog" name="NiveauTraceModules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Niveau d'affichage des modules</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>W:Warning / I:Info / D:Debug</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>251</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pb_select_all">
     <property name="toolTip">
      <string>Activer tous les niveaux d'affichage</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../icons.qrc">
       <normaloff>:/icons/select_all.png</normaloff>:/icons/select_all.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pb_deselect_all">
     <property name="toolTip">
      <string>Inhiber tous les niveaux d'affichage</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../icons.qrc">
       <normaloff>:/icons/remove_all.png</normaloff>:/icons/remove_all.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableWidget" name="table_niveau_aff_modules">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>24</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <row/>
     <column>
      <property name="text">
       <string>Module</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>W1</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>W2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>W3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>I1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>I2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>I3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>D1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>D2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>D3</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item row="0" column="5">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="6">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="7">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item row="0" column="8">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
     <item row="0" column="9">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Unchecked</enum>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
